platform/web/js/libs/library_godot_fetch.js

```diff
@@ -59,7 +59,11 @@ const GodotFetch = {
 			});
 			obj.status = response.status;
 			obj.response = response;
-			obj.reader = response.body.getReader();
+			// `body` can be null per spec (for example, in cases where the request method is HEAD)
+			// As of the time of writing, Chromium (127.0.6533.72) does not follow the spec but Firefox (131.0.3) does
+			// See GH-76825 (godot) for more information
+			// See Chromium revert (of the change to follow the spec): https://chromium.googlesource.com/chromium/src/+/135354b7bdb554cd03c913af7c90aceead03c4d4
+			obj.reader = response.body?.getReader() ?? null;
```

Comment:
Added periods and removed the unnecessary `null` return.

```suggestion
			// `body` can be null per spec (for example, in cases where the request method is HEAD).
			// As of the time of writing, Chromium (127.0.6533.72) does not follow the spec but Firefox (131.0.3) does.
			// See godotengine/godot#76825 for more information.
			// See Chromium revert (of the change to follow the spec): 
			// https://chromium.googlesource.com/chromium/src/+/135354b7bdb554cd03c913af7c90aceead03c4d4
			obj.reader = response.body?.getReader();
```